app/views/comments/_form.html.erb

```diff
@@ -0,0 +1,18 @@
+<%= form_with model: [ @article, @article.comments.build ] do |form| %>
+  <p>
+    <%= form.label :commenter %><br>
+    <%= form.text_field :commenter %>
+  </p>
+  <p>
+    <%= form.label :body %><br>
+    <%= form.text_area :body %>
+  </p>
+  <p>
+    <%= form.submit %>
+  </p>
+  <p>
+    <%= form.label :status %><br>
+    <%= form.select :status, ['public', 'private', 'archived'], selected: 'public' %>
```

Comment:
Could we use the same variable VALID_STATUSES?

Reply:
The VALID_STATUSES variable cannot be used because it's not a class variable and for this case, what is done is to give the options of the type of comment to be published.